sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/changefeed/pkversion/PartitionSynchronizerImpl.java

```diff
@@ -166,10 +169,13 @@ private Flux<Lease> createLeases(Set<String> leaseTokens)
                 return lease;
             })
             .thenMany(Flux.fromIterable(addedLeaseTokens)
-                .flatMap( addedRangeId ->
-                    this.leaseManager.createLeaseIfNotExist(addedRangeId, null), this.degreeOfParallelism)
+                .flatMap( addedRangeId -> {
+                    logger.debug("Adding a new lease document for partition {}", addedRangeId);
```

Comment:
```suggestion
                    logger.info("Adding a new lease document for partition {}", addedRangeId);
```

Reply:
Didn't elevate this for the main branch intentionally
Will change the log level for feature branch